### DISCUSSION TITLE
Welcome to CultOfTheLamb-Truinto Discussions!

### DISCUSSION THREAD

**Discussion by Truinto** [Truinto/CultOfTheLamb-Truinto#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KaicheeBytes** [comment]
I love your mod! So far, I haven't experienced any bugs. However, I would like some clarification regarding the debug check skins function. What does it specifically do?

**Comment by Truinto** [comment]
Great that someone likes it. The debug buttons aren't meant for regular use. I didn't bother removing them after debugging my code. Most of them do nothing or very little. The skin one prints all of the possible skins and variations into a json somewhere (save location probably).

**Reply by KaicheeBytes** [reply]
Oh yeah! I just saw it right before the saves directory.

**Comment by cabalxxvi1985** [comment]
I must be dumb, but how do you use the mod? I can't figure it out for the life of me. It's installed all properly.

**Reply by KaicheeBytes** [reply]
Maybe you haven't set a key bind for it to open? I configured the key bind in the Main Menu of Cult of the Lamb, and when I enter the game through my save, I use my key bind, and it opens up the menu, along with the functions the developer provided. I used Unity Mod Manager since it also offered additional functions, like the follower traits and the doctrines. I also downloaded the zip file and installed it via UMM, but you can also directly unzip the mod in the mods folder.

**Reply by OzaKahuu** [reply]
What is the Keybind?

**Reply by Truinto** [reply]
Alt+F10. Again, only for Unity Mod Manager.

**Comment by OzaKahuu** [comment]
So its not working for me and i haven't the clue why

I've put it in C:\SteamLibrary\steamapps\common\Cult of the Lamb\BepInEx\plugins but there's no mentions of it at all in the logs

Edit: I checked again after some messing around and got it to show and it says that it was "Loading" but does not say if it failed or not

**Comment by rose152** [comment]
I am having issues, the mod hasnt loading into my game and I couldn't open the mod

**Reply by Truinto** [reply]
I see no problem. Make sure you use the write loader (either UMM or Bepinex). If that doesn't help, send me your log file `%appdata%\..\LocalLow\Massive Monster\Cult Of The Lamb\player.log`

**Reply by rose152** [reply]
Hey Truinto, i got the mod to run. Thanks for trying help!

**Reply by OzaKahuu** [reply]
> Hey Truinto, i got the mod to run. Thanks for trying help!

How?

**Comment by hallejr** [comment]
Is it possible to config the mod with the bepinex version? or only the unity mod manager version?

**Reply by Truinto** [reply]
Yes. The downside is you got to edit the settings.json inside the mod's folder. And you have to restart the game to apply the changes.

**Comment by BobBobertBobson** [comment]
I downloaded the Unity mod version to edit my followers traits. I was wondering how to do that. i already made a hot key for the mod nothing really changed.
